lms/templates/seq_module.html

```diff
@@ -13,7 +13,7 @@
     </div>
   % endif
   <div class="sequence-nav">
-    <button class="sequence-nav-button button-previous">
+    <button class="sequence-nav-button button-previous" aria-hidden="true" role="presentation">
```

Comment:
@alisan617 the aria-hidden attribute is enough.  We don't want to say role=presentation since that explicitly says that the element has no semantic value. But since a screen reader user can operate the new tabpanel widget as efficiently has using the Prev and Next buttons, and since those buttons are repeated at the end of the content, it is acceptable to put an aria-hidden here.  Same goes on the change below.